functions/passing-parameters-by-reference.md

```diff
@@ -0,0 +1,98 @@
+# Module name: Passing Parameters by Reference
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+## Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational</td>
+    <td>Avoiding copies using const-reference modifiers</td>
+  </tr>
+  <tr>
+    <td>Main</td>
+    <td>Using references to modify external data</td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td></td>
+  </tr>
+</table>
+
+## Motivation
+
+_Why is this important?_
+_Why do we want to learn/teach this topic?_
+
+## Topic introduction
+
+_Very brief introduction to the topic._
+
+Explain what a reference type is and how it constrasts with a value type.
+
+## Foundational: Using reference types to avoid copies
+
+### Background/Required Knowledge
+
+A student is able to:
+
+* Define and call a function with parameters
+
+### Student outcomes
+
+_A list of things "a student should be able to" after the curriculum._
+_The next word should be an action word and testable in an exam._
+_Max 5 items._
+
+A student should be able to:
+
+1. Use const-refernce types for function arguments
```

Comment:
Shouldn't there first be a "should be able to use reference" before you talk about const-ref?